parsl/monitoring/visualization/views.py

```diff
@@ -56,11 +56,13 @@ def workflow(workflow_id):
                                 % (workflow_id), db.engine)
     task_summary = db.engine.execute(
         "SELECT task_func_name, count(*) as 'frequency' from task WHERE run_id='%s' group by task_func_name;" % workflow_id)
-
+    time_completed = None
+    if hasattr(workflow_details, 'time_completed'):
```

Comment:
What are the circumstances where this `if` statement isn't true?

Reply:
Well, I think I met once before but I was not able to reproduce one for this `if` just now. So I will remove this `if` for now then.

Reply:
Fixed